modin/core/io/column_stores/feather_dispatcher.py

```diff
@@ -52,13 +52,15 @@ def _read(cls, path, columns=None, **kwargs):
             import_optional_dependency(
                 "pyarrow", "pyarrow is required to read feather files."
             )
-            from pyarrow.feather import read_feather
+            from pyarrow import ipc
 
             with OpenFile(
                 path,
                 **(kwargs.get("storage_options", None) or {}),
             ) as file:
-                df = read_feather(file)
-            # pyarrow.feather.read_feather doesn't support columns as pandas.Index
-            columns = list(df.columns)
```

Comment:
`shema.names` already returns a list of columns, so we don't need this explicit conversion anymore